src/components/View.res

```diff
@@ -52,6 +50,8 @@ external make: (
   ~accessibilityLanguage: string=?,
   ~accessibilityLiveRegion: Accessibility.liveRegion=?,
   ~accessibilityRole: Accessibility.role=?,
+  // `role` has precedence over the accessibilityRole prop
+  ~role: Role.t=?,
```

Comment:
When changing View props, you need also to update all place that use the same props.
Can you look for `// rescript-react-native 0.69 View props` and update all the place properly (and bump version number ?)